.gitignore

```diff
@@ -220,3 +220,4 @@ Thunbs.db
 #忽略nuget库 
 packages/
 .vscode/
+/dapr/components
```

Comment:
Why?

Reply:
I used dapr secret component by localfile,so i need ignore this localfile, Now I adjust it 
/dapr/components=>/dapr/localfile